qiskit/qasm2/__init__.py

```diff
@@ -98,8 +115,8 @@
 
 .. autoexception:: QASM2ParseError
 
-Similarly, a failure during the export of an OpenQASM 2 program will raise its own subclass of
-:exc:`QASM2Error`:
+When the exporters fail to export a circuit, likely because it has structure that cannot be
+represented by OpenQASM, they will also emit a custom error.
```

Comment:
```suggestion
represented by OpenQASM 2.0, they will also emit a custom `QASM2ExportError` exception.
```

Reply:
I'll do the first immediately but not the second, since the `QASM2ExportError` is defined literally right under this paragraph and says the same thing - this paragraph is meant to be the lead-in to it.

Reply:
```suggestion
represented by OpenQASM 2.0, they will also emit a custom error.
```